tests/misc/ConstantsTest.py

```diff
@@ -0,0 +1,12 @@
+import unittest
+from coalib.misc import Constants
+
+
+class ConstantsTest(unittest.TestCase):
+
+    def test_default_coafile_presence(self):
+        tmp = Constants.system_coafile
+        Constants.sysytem_coafile = os.path.join(
```

Comment:
yeah , i will do it in my next push . Please check for other things if they are fine or not